.gitignore

```diff
@@ -33,6 +33,8 @@
 
 ## Ignore Visual Studio temporary files, build results, and
 ## files generated by popular Visual Studio add-ons.
+.vscode/
```

Comment:
Changes for this file should be reverted as they are not related to current issue.

Reply:
Reverted. Thank you for the feedback.